tests/benchmarks/test_encoding.py

```diff
@@ -9,7 +9,7 @@
 
 
 msgpack_encoder = RefMsgpackEncoder()
-trace_encoder = MsgpackEncoder()
+trace_encoder = MsgpackEncoder(4 << 20, 4 << 20)
```

Comment:
Are there any reasons in particular in setting the max_size and max_item_size to 4 MB here?

Reply:
This size came up as good enough for holding the data generated during the benchmark runs.

Reply:
`ddtrace/tracer.py:370: error: Cannot determine type of 'dogstatsd'  [has-type]` 🙁